### DISCUSSION TITLE
[Released] Support for Node.js 18.x (LTS)

### DISCUSSION THREAD

**Discussion by rohanrajpal** [vercel/community#986] [Idea]
Edit: https://vercel.com/changelog/node-js-18-lts-is-now-available

---

Nodejs v18 became LTS yesterday, I know its too early to ask, but is there any plan on introducing 18.x runtime soon?

Also I guess Vercel is dependent on Lambda to release v18 first, right? Tracking it here: https://github.com/aws/aws-lambda-base-images/issues/47

**Comment by amyegan** [comment]
You're exactly right @rohanrajpal. We have to wait for upstream providers to enable v18 runtime. Once we have that, Vercel will be able to support Node.js 18

**Comment by limichange** [comment]
Gatsby has just released v5 and the minimum Node.js version required is 18, so we really need it.

**Reply by leerob** [reply]
We are aware and actively working on getting Node.js 18 support!

**Comment by jalandis** [comment]
Node v16 LTS has an updated EOL of September 11th, 2023 in order to match the EOL of its OpenSSL dependency.  Just noting here in case it may alter expected timelines.  

https://nodejs.org/ar/blog/announcements/nodejs16-eol

**Comment by mayrsascha** [comment]
Being able to upgrade to Node v18 would be REALLY nice.
It was released this April.
Its EOL is April 2025 and it's an LTS, meaning I and many other devs wouldn't have to upgrade Node for a long time and instead do something more productive.
It has native support for fetch, can watch for file changes now (bye Nodemon), and has a lot of other goodies.
I hope Vercel starts supporting it soon so my Node upgrade is unblocked.

**Comment by swarnava** [comment]
Node.js 18 is now available for builds and Serverless Functions.
https://vercel.com/changelog/node-js-18-lts-is-now-available

**Reply by PrashantIndurkar** [reply]
Great Thanks for the update
